src/js/discharge-wizard/containers/GuidancePage.jsx

```diff
@@ -1,24 +1,131 @@
+import CarefulConsiderationStatement from '../components/CarefulConsiderationStatement';
+import { branchOfService, board, formNumber } from '../utils';
+import { venueWarning } from '../config';
+
 class GuidancePage extends React.Component {
-  render() {
+  renderResultSummary() {
+    const forReconsideration = this.props.formValues['10_prevApplicationType'] === '3';
+    return `You need to complete Department of Defense (DoD) Form ${formNumber(this.props.formValues)} and send it to the ${board(this.props.formValues).name} for the ${branchOfService(this.props.formValues['7_branchOfService'])}${forReconsideration ? ' for reconsideration' : ''}`;
+  }
+
+  renderOptionalStep() {
+    const validReason = ['1', '2'].indexOf(this.props.formValues['1_reason']) > -1;
+    const dischargeYear = parseInt(this.props.formValues['4_dischargeYear'], 10);
+    const validYear = dischargeYear >= 2001 && dischargeYear <= 2009;
+
+    if (validReason && validYear) {
+      return (
+        <li className="list-group-item">
+          <div>
+            <h4>(Optional): Apply to the Physical Disability Board of Review (PDBR)</h4>
+            <p>
+              The Department of Defense (DoD) created the Physical Disability Board of Review (PDBR) specifically to review appeals about low disability ratings for Veterans discharged between 2001 and 2009.  Many Veterans discharged during this time period were inaccurately given lower disability ratings than they deserved, especially if they suffered from TBI, PTSD, or mental health conditions. If you were discharged in this period and previously received a disability rating for PTSD which you feel is too low, you are eligible to apply to the PDBR for review. The PDBR does not issue discharge status upgrades—but if the PDBR finds that your disability rating was unjustly low, this may help you make your case to upgrade your discharge status. Many Veterans in this situation choose to <a href="http://www.health.mil/PDBR">make a PDBR application</a> before applying for a discharge upgrade. <a href="https://health.mil/Military-Health-Topics/Conditions-and-Treatments/Physical-Disability/Disability-Evaluation/Physical-Disability-Board-of-Review/PDBR-Application-Process">Apply for a PBDR review</a>.
+            </p>
+          </div>
+        </li>
+      );
+    }
+    return null;
+  }
+
+  renderStepOne() {
+    const questionOneResponse = this.props.formValues['1_reason'];
+    const specReason = ['1', '2', '4'].indexOf(questionOneResponse) > -1;
+    const boardToSubmit = board(this.props.formValues);
+    /* eslint-disable quote-props */
+    const reasons = {
+      '1': {
+        name: 'PTSD or other mental health conditions',
+        type: 'condition',
+      },
+      '2': {
+        name: 'TBI',
+        type: 'condition',
+      },
+      '4': {
+        name: 'sexual assault',
+        type: 'experience',
+      },
+    };
+    /* eslint-enable quote-props */
+
+    const strongCaseTips = () => {
```

Comment:
It doesn't seem like this content needs to be in a function as opposed to just being conditionally assigned to a variable.